src/players/seven_reds.py

```diff
@@ -0,0 +1,46 @@
+from typing import Set
+from outcome import Outcome
+from .martingale import Martingale
```

Comment:
use absolute import instead of relative import. [absolute vs relative import](https://realpython.com/absolute-vs-relative-python-imports/)

Reply:
Done.